### ISSUE TITLE
Question about castRadiance function

### ISSUE THREAD

**Issue by tdkr** [SabakiHQ/influence#4]
```javascript
if (d >= p1 || data[n[1]] && data[n[1]][n[0]] == null && data[n[1]][n[0]] === -sign || n in visited)
    continue
```
A question about above code, isn't 'data[n[1]] && data[n[1]][n[0]] == null && data[n[1]][n[0]] === -sign' always return false?

**Comment by yishn** [comment]
Nice catch! Looking at the commit history, it's probably supposed to be instead:

~~~js
if (d >= p1 || data[n[1]] && data[n[1]][n[0]] != null && data[n[1]][n[0]] === -sign || n in visited)
~~~

or equivalently just:

~~~js
if (d >= p1 || data[n[1]] && data[n[1]][n[0]] === -sign || n in visited)
~~~

The middle condition is supposed to block radiance if encountering an enemy stone. Do you mind creating a PR for this?

**Comment by tdkr** [comment]
> Nice catch! Looking at the commit history, it's probably supposed to be instead:
> 
> ```js
> if (d >= p1 || data[n[1]] && data[n[1]][n[0]] != null && data[n[1]][n[0]] === -sign || n in visited)
> ```
> 
> or equivalently just:
> 
> ```js
> if (d >= p1 || data[n[1]] && data[n[1]][n[0]] === -sign || n in visited)
> ```
> 
> The middle condition is supposed to block radiance if encountering an enemy stone. Do you mind creating a PR for this?

Sure, thanks for your reply

**Comment by yishn** [comment]
Closed with #5.
